conan/tools/apple/apple.py

```diff
@@ -194,26 +204,33 @@ def fix_apple_shared_install_name(conanfile):
     *install_name_tool* utility available in macOS to set ``@rpath``.
     """
 
+    if not is_apple_os(conanfile):
+        return
+
+    xcrun = XCRun(conanfile)
+    otool = xcrun.otool
+    install_name_tool = xcrun.install_name_tool
```

Comment:
Alternatively, we could just prefix all of the commands with `xcrun`, e.g. `xcrun otool -hv {file}`